notebook/tests/launchnotebook.py

```diff
@@ -39,11 +39,12 @@ class NotebookTestBase(TestCase):
 
     port = 12341
     config = None
+    url_prefix = '/a%40b/'
```

Comment:
Ah I just read the original issue that this PR closes.  You're doing this just to test our ability to handle the @ symbol in the path name!